#! /usr/bin/env python

PACKAGE='tuw_marker_playground'
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("show_debug_image", bool_t, 0, "Show camera debug image", True)
gen.add("marker_dot_displacement", double_t, 0, "The outer dots are displaced by this value on the x and y axes.", 0.06)
gen.add("publish_fiducials", bool_t, 0, "Publish fiducials to allow pose estimation.", True)
exit(gen.generate(PACKAGE, "tuw_marker_playground", "Dice"))
